datasync_node:
  ros__parameters:
    # DAVIS 240 defaults for this bag
    weight_param: 240
    height_param: 180
    focus: 3683.2
    pixel_size: 18.5

    # For this bag, IMU lags behind events
    require_imu_ahead: false
    # IMU averaging window (nanoseconds)
    imu_window_ns: 8000000

    events_topic: "events"
    imu_topic: "imu"
    count_image_topic: "count_image"
    time_image_topic: "time_image"
    publish_time_image: true
    time_image_vis_topic: "time_image_vis"
    publish_time_image_vis: true
    publish_raw_count: true
    raw_count_topic: "count_image_raw"
    sort_events_by_time: true
